### DISCUSSION TITLE
Submission format of scrolly content

### DISCUSSION THREAD

**Discussion by ran-codes** [Drexel-UHC/uhc-graphics-pdph-life-expectancy-explainer#4] [General]
@ascheeres @matodd-beep @usamabilal @anujtanwar98 

Here is an example of how you can format your scrolly so it is easy for the developers to build. Feel free to put it in an excel sheet or any table for us.

## Example

<html>
<body>

Text (WITH EACH CLICK] | Image
-- | --
Introduction
Illustrate mortality crisis in U.S. Define reproductive justice Explain how abortion access and paid family leave (PFL) are foundational for reproductive justice and how restrictions can perpetuate mortality crisis and racial disparities |  
The U.S. is the only Organization for Economic Co-operation and Development (OECD) member country that does not guarantee access to paid time off for childbirth. | Zoom in on U.S. map
Though the federal Family and Medical Leave Act (FMLA) provides 12-weeks of protected yet unpaid time for family or medical reasons, strict eligibility requirements result in only 56% of the workforce being eligible. | Fill in 56% of map
Employers fill some of this gap, but lower-wage and minoritized workers are less likely to have access to these benefits. And some state governments prevent city and county local governments from enacting PFL policies.  As a result, families are subject to a patchwork of employer, local, and state government policies that leave millions of pregnant people, who are disproportionately minoritized and low-income individuals, without access to PFL. | Highlight remaining 44%
On June 24, 2022 the U.S. Supreme Court overturned Roe v. Wade, giving states the authority to ban or restrict abortion access. | Highlight entire country to indicate state-by-state authority
This legal landscape has left it possible for these restrictions overlap, with state legislators and governors effectively forcing people to give birth and return to work or risk losing pay. | Text: How many states restrict both abortion access and paid family leave?
PFL Landscape
  | Clear map
X states mandate paid protected time off for family and medical reasons, including birth and care of a newborn or adoption. | Highlight these states in PFL1 color
X states have no paid family leave and do not prevent cities or counties from offering it. | Highlight these states in PFL2 color
X states offer no paid family leave and prevent city or county governments from offering it. | Highlight these states in PFL3 color
Abortion Access Landscape
  | Clear map
Abortion is currently accessible in X states. | Highlight these states in AB1 color
X states have laws that restrict abortion care or have attempted to ban abortion. | Highlight these states in AB2 color
X states have banned abortion. | Highlight these states in AB3 color
Analysis
Here’s what we found. | Clear map
None of the states that ban or restrict abortion offer paid family leave. That’s half of the states in this country. | Highlight of states that ban or restrict abortion (AB2, AB3), add PFL2, PFL3 to indicate blend
16 of 25 states that ban or are hostile to abortion prevent local governments from enacting their own PFL policies. | Highlight AB2, AB3, add PFL3
Of the 25 states and DC where abortion remains accessible, 12 have PFL policies. | Highlight AB1, add PFL1
Conclusion, CTA |  
Looking at the map, it becomes clear why there’s a maternal health crisis in the U.S.  Offer details and call to action   [End here?]  These policy decisions represent a profound injustice -- forcing parenthood and then leaving parents without support after childbirth – they also reveal patterns. | [Ran: End here, or keep going with analysis below?]
The 25 states with abortion restrictions and bans and no PFL reflect historical and contemporary patterns of institutionalizing racism via policy. Together, these policies constitute a form of structural racism and reproductive oppression because they amplify the impacts of racially inequitable policy structures that disproportionately impact Black pregnant people, thus perpetuating racial and ethnic inequities in infant and maternal health outcomes. |  
These policies are geographically and politically patterned: the 16 states that preempt PFL and restrict abortion access are all located in the Midwest and South, and nearly all have Republican governors and legislatures. These states, particularly those in the South, are home to large Black populations, and lack additional policies to support pregnant people and families, including postpartum healthcare access through Medicaid, a living wage, or paid sick leave, all of which disproportionately impact Black and other birthing people of color6. |  

</body>
</html>Text (WITH EACH CLICK]

**Comment by usamabilal** [comment]
Some ideas here. @matodd-beep do you think this is too complex?

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/ub45/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/ub45/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



Text | image
-- | --
Intro |  
Something on how we measure how long   people live, one measure is life expectancy |  
Definition of life expectancy   (placeholder) |  
Trends |  
Evolution over time in the US: some   description of overall trends | Long term LE time trends in the US (from NCHS or?)
Mortality crises: zoom in to 1918, WW2,   HIV, COVID | Zoom to each one (4 steps)
Evolution by state: these changes have   not been homogeneous | Time trends with all the states, highlight top (CA/NY?) and bottom (WV?),   but also PA
Evolution in Philadelphia | Time trends with just Philadelphia
Mortality crises in Philadelphia (COVID;   do we have data far enough to capture HIV?) |  
Disparities by race/ethnicity |  
Back to 2015-2019 (for example): lets   compare racial/ethnic disparities in LE | bar graph with LE by race/ethnicity
Changes in covid by race/ethnicity | Time trend with one line per race, show what happened in 2020 and 2021
Disparities by space (zip   code?) |  
Back to 2015-2019 (or whatever is   avialable): LE is heterogeneous in philly | Map with LE by zip code
Changes in covid by zip code | Time trend with one line per race, show what happened in 2020 and 2021   (possible?)
Some explanation for factors driving   racial/ethnic and spatial disparities | Maybe a figure with a conceptual framework
Causes of death driving LE |  
Contribution of specific causes of death   to LE (deleted cause LE? Or arriaga decomp?) | Bar chart?
Contribution of specific causes of death   to LE (deleted cause LE? Or arriaga decomp?) by sex | Bar chart with sexes
Contribution of specific causes of death   to LE (deleted cause LE? Or arriaga decomp?) by race/ethnicity | bar chart with the different race/ethnicities
Contribution of specific causes of death   to LE (deleted cause LE? Or arriaga decomp?) pre/post covid | bar chart with the contribution by year



</body>

</html>

**Reply by ran-codes** [reply]
are we still thinking about splitting into three scrollies as per the [first discussion](https://github.com/Drexel-UHC/uhc-graphics-pdph-life-expectancy-explainer/discussions/2)?

**Reply by usamabilal** [reply]
Good point Ran. Yes, we can "modularize" this. Each section (marked by the text without an image) would be its own scrolly for now. 
LEts start with Trends, since @ascheeres has more data on that i think.

**Comment by ran-codes** [comment]
From @ascheeres 

> Hi Usama,
> 
> I just had a chance to review the draft table and think it looks great! I have a few thoughts/answers to questions you posed:
> 
> - For the “Trends” section:
>    -  I did some digging and found 1900-2018 LE estimates by sex and race from NCHS: [https://www.cdc.gov/nchs/data-visualization/mortality-trends/index.htm](https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.cdc.gov%2Fnchs%2Fdata-visualization%2Fmortality-trends%2Findex.htm&data=05%7C01%7Crl627%40drexel.edu%7C28a72398485e4caf91c908db978ea5b5%7C3664e6fa47bd45a696708c4f080f8ca6%7C0%7C0%7C638270411141119037%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kQAKylaao4MLcqoO%2BkMu3Z2oCjtxYPXmK48nr%2BhNsV0%3D&reserved=0). This csv file only includes estimates for all races, Black, and white, but starting in 2006, more recent LE estimates (through 2020) by sex and race/Hispanic ethnicity are available here: [https://www.cdc.gov/nchs/products/life_tables.htm#life](https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.cdc.gov%2Fnchs%2Fproducts%2Flife_tables.htm%23life&data=05%7C01%7Crl627%40drexel.edu%7C28a72398485e4caf91c908db978ea5b5%7C3664e6fa47bd45a696708c4f080f8ca6%7C0%7C0%7C638270411141119037%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=PQKoEn49S9nF5LpeupV8saw1THLKnZ7Ur0kkamGqAII%3D&reserved=0). Though less ideal than csv files, we could work with PDF files.
>    - I found 2018-2020 LE estimates by state available for download from the CDC: [https://www.cdc.gov/nchs/pressroom/sosmap/life_expectancy/life_expectancy.htm](https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.cdc.gov%2Fnchs%2Fpressroom%2Fsosmap%2Flife_expectancy%2Flife_expectancy.htm&data=05%7C01%7Crl627%40drexel.edu%7C28a72398485e4caf91c908db978ea5b5%7C3664e6fa47bd45a696708c4f080f8ca6%7C0%7C0%7C638270411141119037%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=u0vHQKVqEj02SgZjvv99XdrugQEdPXLcD8jDJG7wfQE%3D&reserved=0). The supplemental material for this paper includes LE estimates by state from 1959-2016: [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7146991/](https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7146991%2F&data=05%7C01%7Crl627%40drexel.edu%7C28a72398485e4caf91c908db978ea5b5%7C3664e6fa47bd45a696708c4f080f8ca6%7C0%7C0%7C638270411141119037%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2FEaTvnuBtBkhkEwmxsko4Vc64hmNsqka1GaajY%2F544Y%3D&reserved=0). Again, though not ideal, we could work with this PDF file.
>     - We have enough data to capture HIV mortality rates by sex and for non-Hispanic Black Philadelphians (estimates are mostly suppressed for other racial/ethnic groups).
> - For the spatial disparities section, we should be able to calculate LE estimates by ZIP code.
>  - I’ve never estimated the contribution of specific causes of death to LE, so I think @matodd-beep should weigh in on whether the deleted cause or Arriaga decomposition methods are more appropriate.
